sys/net/network_layer/ng_sixlowpan/frag/ng_sixlowpan_frag.c

```diff
@@ -80,18 +80,19 @@ static uint16_t _send_1st_fragment(ng_sixlowpan_netif_t *iface, ng_pktsnip_t *pk
                                    size_t payload_len, size_t datagram_size)
 {
     ng_pktsnip_t *frag;
-    uint16_t max_frag_size = _floor8(iface->max_frag_size -
-                                     (payload_len - datagram_size) -
-                                     sizeof(ng_sixlowpan_frag_t));
     uint16_t local_offset = 0;
+    /* payload_len: actual size of the packet vs
+     * datagram_size: size of the uncompressed IPv6 packet */
+    int payload_diff = (datagram_size - payload_len);
```

Comment:
Yes and no. It would be wrong to call it that, since with IPHC you don't really have a `dispatch_len` anymore as you had in the line you were referring to. In the case of IPHC it is basically `sizeof(ipv6_hdr_t) - len(iphc_dispatch)`. For UNCOMP you are right.